### ISSUE TITLE
Add Gradle

### ISSUE THREAD

**Issue by rdrpenguin04** [JonahSloan/NDSROM#4]
It would be nice to use something other than an IDE for this.

This is more a tracking issue than anything; I am working on this as I make this issue.

**Comment by rdrpenguin04** [comment]
It turns out this may actually be hard. I can't even find a main class.

**Comment by Lonami** [comment]
Yeah I took a look and most things don't seem implemented yet. It's mostly classes to (presumably) model the way ROMs work along with a byte reader/writer.

**Comment by Jenna1337** [comment]
Yeah, I wasn't really sure on how to implement it, and some DS ROMs store some data in strange locations.
